### PR TITLE
Adds support for repository to be an output, Adding in test for #133

### PR DESCRIPTION
**Pull request by rshade**
This adds in a test for #133, with image arguments. It also sets the chart version to match the image version.

### CODE REVIEW

**Comment by rshade** [comment]
I pinned google-protobuf because of this issue: https://github.com/pulumi/pulumi/issues/17421

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# EditorConfig helps maintain consistent coding styles across editors and IDEs
+# More details at https://editorconfig.org
```
Comment (blampe) [inline .editorconfig:2]: Interesting. Eventually I'd like ci-mgmt to ship some commit hooks to apply consistent linting and formatting for us.

**Review by blampe — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (rshade) [reply .editorconfig:2]: That would be great. I love editorconfig. It allows, it allows vscode to just know what to use automagically. We could ship it via ci-mgmt, like we do go-release.

Comment (rshade) [reply .editorconfig:2]: I added it because the default to .ts is 4spaces in my vscode and it kept refactoring files.

**Pull request merged**